packages/frontend/src/components/markdown-container/markdown-container.tsx

```diff
@@ -36,6 +37,7 @@ import { isHashUrl, isRelativeUrl } from '../../shared/url-utils';
 import { pick } from '../../shared/utility';
 
 import { ChakraUIRenderer } from './chakra-ui-renderer';
+import { IexMarkdownSchema } from './iexMarkdownSchema';
```

Comment:
Can you convert this to `iex-markdown-schema` to stick with the file naming convention